lasagne/init.py

```diff
@@ -50,6 +106,45 @@ def sample(self, shape):
 
 
 class Glorot(Initializer):
+    """Glorot weight initialization [1]_
+
+    This is also known as Xavier initialization.
+
+    Parameters
+    ----------
+    initializer : lasagne.init.Initializer
+        Initializer used to sample the weights. The default is Gaussian.
```

Comment:
Yeah, `Uniform` as default makes more sense. Although people should really be using `GlorotUniform` and not `Glorot` directly. We could also just get rid of the defaults (for both `Glorot` and `He`), I guess. I'm fine with whatever you guys decide :)